package.json

```diff
@@ -58,6 +58,7 @@
     "simple-git-hooks": "^2.8.0",
     "ts-node": "^10.9.1",
     "typescript": "~4.6.3",
+    "web3-provider-engine": "^16.0.3",
```

Comment:
It seems that if you want to augment a type and you need to grab a type from another package within that augmentation, you need to declare the dependency in the root, otherwise ESLint will complain.

Reply:
Interesting, OK